CHANGELOG.md

```diff
@@ -1,10 +1,3 @@
-## 5.0.0-beta.5
-### Breaking Changes
-* iOS rename `productCategory` to `product type` and `productType` to `productSubtype` (#377) via Josh Holtz (@joshdholtz)
-* Android rename `productCategory` to `productType` and `productType` to `productSubtype` (#376) via Josh Holtz (@joshdholtz)
-### Other Changes
-* Use new mapped product type values for `purchaseProduct()` and `getProductInfo()` (#384) via Josh Holtz (@joshdholtz)
```

Comment:
version 6's changelog should include what was released in version 5, so you might not want to revert this